website/www/site/content/en/contribute/_index.md

```diff
@@ -57,46 +57,98 @@ There are lots of opportunities to contribute. You can for example:
+  [Beam PMC](https://home.apache.org/phonebook.html?pmc=beam) members should mark their votes as binding.
+
+### Communication
+
+All communication is expected to align with the [Code of Conduct](https://www.apache.org/foundation/policies/conduct).
+
+Discussion about contributing code to Beam happens on the [dev@beam.apache.org mailing list](/community/contact-us/). Introduce yourself!
+
+Questions can be asked on the [#beam channel of the ASF slack](https://beam.apache.org/community/contact-us/). Introduce yourself!
+
+<a class="arrow-list-header" data-toggle="collapse" href="#collapseContributing" role="button" aria-expanded="false" aria-controls="collapseContributing">
+
+{{< figure src="/images/arrow-icon_list.svg">}}
+
+## Before You Begin
+
+</a>
 
 ### Prerequisites
 
- - A GitHub account.
- - A Linux, macOS, or Microsoft Windows development environment
- - Java JDK 8 installed
- - [Docker](https://www.docker.com/) installed for some tasks including building worker containers and testing this website
-   changes locally.
- - For SDK Development:
-      - [Go](https://golang.org) 1.12 or later installed for Go SDK development
-      - Python 3.x interpreters. You will need Python interpreters for all Python versions supported by Beam.
-        Interpreters should be installed and available in shell via `python3.x` commands. For more information, see:
-        Python installation tips in [Developer Wiki](https://cwiki.apache.org/confluence/display/BEAM/Python+Tips#PythonTips-InstallingPythoninterpreters).
- - For large contributions, a signed [Individual Contributor License
-   Agreement](https://www.apache.org/licenses/icla.pdf) (ICLA) to the Apache
-   Software Foundation (ASF).
-
-
-### Configuration options
+- A [GitHub](https://github.com/) account.
+- A Linux, macOS, or Microsoft Windows development environment
+- Java JDK 8 installed
+- [Docker](https://www.docker.com/) installed for some tasks including building worker containers and testing this website
+  changes locally.
+- For SDK Development:
+   - [Go](https://golang.org) 1.12 or later installed for Go SDK development
+   - Python 3.x interpreters. You will need Python interpreters for all Python versions supported by Beam.
+     Interpreters should be installed and available in shell via `python3.x` commands. For more information, see:
+     Python installation tips in [Developer Wiki](https://cwiki.apache.org/confluence/display/BEAM/Python+Tips#PythonTips-InstallingPythoninterpreters).
+- For large contributions, a signed [Individual Contributor License
+  Agreement](https://www.apache.org/licenses/icla.pdf) (ICLA) to the Apache
+  Software Foundation (ASF).
+
+### Share Your Intent
+1. Find or create an issue in the [Beam issue tracker (JIRA)](https://issues.apache.org/jira/projects/BEAM/issues).
+   Tracking your work in an issue will avoid duplicated or conflicting work, and provide
+   a place for notes. Later, your pull request will be linked to the issue as well.
```

Comment:
Add: 
You may find the [Jira beginner’s guide](https://cwiki.apache.org/confluence/display/BEAM/Beam+Jira+Beginner%27s+Guide) and some useful [Jira tips](https://cwiki.apache.org/confluence/display/BEAM/Jira+Tips) in our Wiki.

Reply:
done